lib/driver.js

```diff
@@ -1118,8 +1118,22 @@ class XCUITestDriver extends BaseDriver {
 
     if (this.isRealDevice()) {
       await installToRealDevice(this.opts.device, this.opts.app, this.opts.bundleId, this.opts.noReset);
+      if (this.opts.otherApps) {
+        log.warn('Capability otherApps is only supported for Simulators');
```

Comment:
Any special reason to disable this feature for real device?
For example, to avoid issues about signing for otherApps (and fail to install them)

Reply:
Yes, I wanted to avoid real devices for now due to signing issues. What do you guys think?

Reply:
I think only alternative would be making it abundantly clear that the users are responsible for making sure signing is all a-ok. Seems like a capital-H Headache. :(

Reply:
Let's start with only for simulators and see if we have requests to add this for real devices. What do you guys think?

Reply:
Sound reasonable, for me.
Then, how about adding something like "please create a feature request on appium repo if you want this feature for real device"? Technically, we can call installToRealDevice.

Reply:
Sounds good.